eduid_dashboard_amp/__init__.py

```diff
@@ -14,17 +14,18 @@
     # TODO: Arrays must use put or pop, not set, but need more deep refacts
     'norEduPersonNIN',
     'eduPersonEntitlement',
-    'mobile',
+    'phone_numbers',
```

Comment:
I believe the mongo doc key should be 'phone', not 'phone_numbers', to match where it is parsed in user._parse_phone_numbers():

https://github.com/SUNET/eduid-userdb/blob/master/src/eduid_userdb/user.py#L183